.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore csv files
+*.csv
+
+# Ignore data folder with scraped and saved tables
+data/
```

Comment:
It seems a bit redundant: .csv files are expected to be data files hence in `data/` folder